hapless/main.py

```diff
@@ -340,3 +357,17 @@ def signal(self, hap: Hap, sig: signal.Signals):
                 f"{config.ICON_INFO} Cannot send signal to the inactive hap",
                 style=f"{config.COLOR_ERROR} bold",
             )
+
+    def restart(self, hap: Hap):
+        hid, name, cmd = hap.hid, hap.name, hap.cmd
+
+        if hap.active:
+            self.kill([hap], verbose=False)
+
+        hap_killed = self.get_hap(str(hid))
```

Comment:
no need to `str`, I think it's a wrong type hint for the hid return type `-> int`